frontend/ui/lib/widgets/create_trip_page_widget.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:tripmeout/widgets/create_trip_widget.dart';
+import 'package:tripmeout/widgets/map_widget.dart';
+
+class CreateTripPageWidget extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+        child: FittedBox(
+      child: Column(
+        children: <Widget>[
```

Comment:
nit: <Widget> isn't actually necessary here (or on any of the `children:` statements). The compiler will infer it.

Reply:
(The `<Widget>` type parameter)